shark-graph/src/main/java/shark/internal/HprofInMemoryIndex.kt

```diff
@@ -243,7 +244,7 @@ internal class HprofInMemoryIndex private constructor(
     /**
      * Class ids for primitive wrapper types
      */
-    private val primitiveWrapperTypes = mutableSetOf<Long>()
+    private val primitiveWrapperTypes = LongScatterSet()
```

Comment:
Did this have a meaningful impact?

Reply:
I really need to come up with way of reliable measurements. Thinking about trying `benchmark` lib..

Will measure these in isolation, but my question here is: do you think there's harm to doing this?

Reply:
A bit, in that it increases our usage of these custom classes that we may want to ditch one day. We should only use those when we know we need them.